pkg/operator/controller/ingress-routes/controller.go

```diff
@@ -0,0 +1,250 @@
+package ingress
+
+import (
+	"context"
+	"fmt"
+
+	logf "github.com/openshift/cluster-ingress-operator/pkg/log"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	configv1 "github.com/openshift/api/config/v1"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+const (
+	controllerName = "ingress_routes_controller"
```

Comment:
Would you be opposed to using a different controller name? Maybe `operator_route_rbac_controller`, or something along those lines? I think `ingress_routes_controller` is a little vague and somewhat similar to the existing `ingress-to-route` controller in OCM.

Reply:
That sounds good to me, I'll include the change in the next update to the PR.

Reply:
Could we change the `controller/` sub dir for this controller as well? ie
`pkg/operator/controller/operator-route-rbac-controller`?

Reply:
I'll make sure to update the directory name.